### PR TITLE
45056 Create project solution for Maritime Safety Information

### PR DESCRIPTION
**Pull request by Shirin14926**
Create Project skeleton.
 Unit test project added

Note: Referred PBI 45721 for Story Points

### CODE REVIEW

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Controllers/HomeController.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        }
+
+        public IActionResult Index()
+        {
+            return View();
+        }
+
+        public IActionResult Privacy()
+        {
+            return View();
```
Comment (rockydevnet) [inline UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Controllers/HomeController.cs:23]: This view doesn't exist?

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web.Tests/UKHO.MaritimeSafetyInformation.Web.Tests.csproj [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>enable</Nullable>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.11.0" />
```
Comment (rockydevnet) [inline UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web.Tests/UKHO.MaritimeSafetyInformation.Web.Tests.csproj:11]: Could we make sure that we start with the latest version of the packages please. (.e. Test.Sdk should be 17.1.0, NUnit 3.3.13, NUnitTestAdaptor 4.2.1, etc).

**Review by rockydevnet — CHANGES_REQUESTED** [review]

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Controllers/HomeController.cs (hunk 1 above)
Comment (Shirin14926) [reply UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Controllers/HomeController.cs]: Removed, Thanks Martin

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web.Tests/UKHO.MaritimeSafetyInformation.Web.Tests.csproj (hunk 2 above)
Comment (Shirin14926) [reply UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web.Tests/UKHO.MaritimeSafetyInformation.Web.Tests.csproj]: Added latest versions packages. Thanks

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Program.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
+WebApplicationBuilder? builder = WebApplication.CreateBuilder(args);
```
Comment (barrie-cooper) [inline UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Program.cs:1]: Don't think this needs to be nullable:
```suggestion
WebApplicationBuilder builder = WebApplication.CreateBuilder(args);
```

Comment (Shirin14926) [reply UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Program.cs]: Done. Thanks Barrie

**Review by usha13529 — APPROVED** [review]
LGTM.

**Review by barrie-cooper — APPROVED** [review]
LGTM

**Review by rockydevnet — APPROVED** [review]
LGTM

**Pull request merged**